lib/system/cgprocs.nim

```diff
@@ -13,8 +13,8 @@ type
   LibHandle = pointer       # private type
   ProcAddr = pointer        # library loading and loading of procs:
 
-proc nimLoadLibrary(path: string): LibHandle {.compilerproc, hcrInline, nonReloadable.}
-proc nimUnloadLibrary(lib: LibHandle) {.compilerproc, hcrInline, nonReloadable.}
-proc nimGetProcAddr(lib: LibHandle, name: cstring): ProcAddr {.compilerproc, hcrInline, nonReloadable.}
+proc nimLoadLibrary(path: string): LibHandle {.compilerproc, nonReloadable.}
```

Comment:
Personally remove `nonreloadable`, as well.

I know it increases the size, but it more importantly highlights how invasive this whole feature was in the first place.

Reply:
this is non-blocking, I'm good to merge it.

Reply:
It can go too - the required changes are of an okay amount.

Reply:
Sweet!